### PR TITLE
Create cromwell-version.conf

### PR DESCRIPTION
**Pull request by ruchim**
Happy to remove Wdl4s versioning if that's not anything us or a user would find helpful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3115,6 +3116,33 @@ Response:
...
+
+Response:
+```
+"date": "Sun, 18 Sep 2016 14:38:11 GMT",
+"server": "spray-can/1.3.3",
+"content-length": "33",
+"content-type": "application/json; charset=UTF-8"
+
+{
+  "cromwell": 23,
```
Comment (ruchim) [inline README.md]: Need to fix this, it will actually be 23-xxxxxx-SNAP

File: engine/src/main/resources/swagger/cromwell.yaml [hunk 2]
```diff
@@ -822,3 +842,12 @@ definitions:
...
         description: The number of currently running jobs
+  VersionResponse:
+    description: 'Provides the version of the engine running inside the system'
+    properties:
+      cromwell:
+        type: string
+        description: The version of the Cromwell Engine
+      wdl4s:
+        type: string
+        description: The version of the Wdl4S beign used by Cromwell
```
Comment (Horneth) [inline engine/src/main/resources/swagger/cromwell.yaml]: Do we really want the wdl4s version ? wdl4s is just a dependency of cromwell as many others IMO.
The wdl version (when well defined) would be better.

Comment (Horneth) [inline engine/src/main/resources/swagger/cromwell.yaml]: Also beign -> being

File: engine/src/main/resources/swagger/cromwell.yaml [hunk 3]
```diff
@@ -822,3 +842,12 @@ definitions:
       jobs:
         type: integer
         description: The number of currently running jobs
+  VersionResponse:
+    description: 'Provides the version of the engine running inside the system'
```
Comment (mcovarr) [inline engine/src/main/resources/swagger/cromwell.yaml]: I think the outer quotes are superfluous?  Also maybe "Returns the version of Cromwell"

File: project/Version.scala [hunk 4]
```diff
@@ -19,6 +19,20 @@ object Version {
...
+      s"""|version {
+          |  cromwell: "${version.value}"
+          |  wdl4s: "${Dependencies.wdl4sV}"
+          |}
+          |""".stripMargin
+    IO.write(file, contents)
+    Seq(file)
+  }
+
+  val versionConfCompileSettings = List(resourceGenerators in Compile <+= writeVersionConf)
```
Comment (mcovarr) [inline project/Version.scala]: careful, you could put an eye out with that operator

File: engine/src/main/resources/swagger/cromwell.yaml (hunk 2 above)
Comment (mcovarr) [inline engine/src/main/resources/swagger/cromwell.yaml]: lowercase `wdl4s`

File: engine/src/main/scala/cromwell/webservice/CromwellApiService.scala [hunk 5]
```diff
@@ -271,6 +271,21 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
       }
     }
 
+  def versionRoute =
+    path("engine" / Segment / "version") { version =>
+      get {
+        complete {
+          val versionConf = ConfigFactory.load("cromwell-version.conf").getConfig("version")
```
Comment (mcovarr) [inline engine/src/main/scala/cromwell/webservice/CromwellApiService.scala]: maybe `lazy val` this as a class member so it doesn't have to be loaded every time somebody hits the endpoint

**Comment by mcovarr** [comment]
:+1:

File: project/Version.scala (hunk 4 above)
Comment (ruchim) [inline project/Version.scala]: 🙈

**Comment by Horneth** [comment]
👍

**Pull request merged**